src/style.css

```diff
@@ -0,0 +1,117 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: auto;
+}
+
+.card__product-name {
+  display: flex;
```

Comment:
```suggestion
```
this parameter is unnecessary here. I suppose everything is going to perform well without it.